lib/puma/runner.rb

```diff
@@ -64,30 +63,19 @@ def start_control
       control.min_threads = 0
       control.max_threads = 1
 
-      case uri.scheme
```

Comment:
Wait this is possible? Can we do this is a separate PR?

Reply:
Extracted this change to #2111.